lib/codecs/src/decoding/format/influxdb.rs

```diff
@@ -168,9 +167,11 @@ mod tests {
     #[test]
     fn deserialize_success() {
         let deser = InfluxdbDeserializer::new(true);
-        let buffer = Bytes::from(
-            "cpu,host=A,region=west usage_system=64i,usage_user=10i 1590488773254420000",
-        );
+        let now = chrono::Utc::now();
```

Comment:
Refactored this a little bit, If you prefer how it was before, I could just change `DateTime::from_timestamp_micros` to `DateTime::from_timestamp_nanos` in this test

Reply:
I personally prefer the hardcoded input, but it's not a big deal 👍